idl/storage_service.idl.hh

```diff
@@ -59,8 +59,15 @@ struct raft_topology_snapshot {
 
 struct raft_topology_pull_params {};
 
+struct raft_general_snapshot {
+    utils::chunked_vector<canonical_mutation> auth_mutations;
```

Comment:
When I was thinking about generic snapshot pull rpc I was thinking about an RPC that takes table name as a parameter and pulls its mutations. This way nothing have to be extended when other tables move to group0 a snapshot pulling code just needs one more call.

Reply:
ok, makes sense too, but why topology wasn't done like this?

Reply:
Because nobody expected that new tables will be added to topology snapshot when it was added. But when we started adding them it somewhat made sense to put it into the topology snapshot because they were related and not we start pulling thing that are not topology related. May be we should move topology snapshot to this new RPC indeed. It is not too late.

Reply:
ok will add table(s) name to RPC, 
removal of topology snapshot can be addressed in separate PR, if as I understand it wasn't released yet.

Reply:
> removal of topology snapshot can be addressed in separate PR

Of course.